analysis.tex

```diff
@@ -93,13 +93,100 @@ \section{Analysis}\label{sec:analysis}
 definition of computational indistinguishability is included in
```

Comment:
The complete proofs are not in the Appendix any more "for the complete proofs see Appendix".

Reply:
fixed